server/api/users/me-auth/ctftime/patch.js

```diff
@@ -0,0 +1,34 @@
+const { responses } = require('../../../../responses')
+const database = require('../../../../database')
+const auth = require('../../../../auth')
+
+module.exports = {
+  method: 'patch',
```

Comment:
This should be an HTTP `PUT`, since it can also create a new ctftime auth, and it is already idempotent.